BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/physicsOfResultsPUEO.py

```diff
@@ -32,13 +33,16 @@
 parser.add_argument("energy", help="energy of neutrino", type=int)
 g = parser.parse_args()
 
+print(g.source, g.destination, g.indiv, g.energy)
+
 libPath = "/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib"
 #libPath = "/users/PAS1960/dylanwells1629/buildingPueoSim/pueoBuilder/lib"
 
-ROOT.gSystem.Load(libPath + "/libNiceMC.so")
-ROOT.gSystem.Load(libPath + "/libAntarcticaRoot.so")
-ROOT.gSystem.Load(libPath + "/libAnitaEvent.so")
-ROOT.gSystem.Load(libPath + "/libPueoSim.so")
+print(libPath + "/libNiceMC.so")
+ROOT.gSystem.Load("/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/libNiceMC.so")
+ROOT.gSystem.Load("/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/libAntarcticaRoot.so")
+ROOT.gSystem.Load("/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/libAnitaEvent.so")
+ROOT.gSystem.Load("/fs/ess/PAS1960/buildingPueoSim/pueoBuilder/lib/libPueoSim.so")
```

Comment:
What's the reason for reverting this change?

Reply:
Oh, I was running into errors about files not existing, so I was testing to see if it was causing the issue. It looks like I forgot to revert it back when I found the actual issue - I'll do that now.